src/current/_includes/molt/molt-docker.md

```diff
@@ -0,0 +1,78 @@
+### Performance
+
+MOLT Fetch and Verify are likely to run more slowly in a Docker container than on a local machine. To improve performance, increase the memory or compute resources on your Docker container.
+
+{% if page.name == "molt-fetch.md" %}
+### Authentication
+
+{{site.data.alerts.callout_info}}
+It is only necessary to specify volumes and environment variables when using MOLT Fetch with [cloud storage](#cloud-storage), as described in the following sections. No additional configuration is needed when running MOLT Fetch with a [local file server](#local-file-server) or in [direct copy mode](#direct-copy): 
+
+~~~ shell
+docker run -it molt fetch ...
+~~~
+
+For more information on `docker run`, see the [Docker documentation](https://docs.docker.com/reference/cli/docker/container/run/).
+{{site.data.alerts.end}}
+
+#### Google Cloud Storage
+
+If you are using [Google Cloud Storage](https://cloud.google.com/storage/docs/access-control) for [cloud storage](#cloud-storage):
+
+Volume map the `application_default_credentials.json` file into the container, and set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to point to this file.
+
+~~~ shell
+docker run \
+  -v ~/.config/gcloud/application_default_credentials.json:/gcp/creds.json:ro \
+  -e GOOGLE_APPLICATION_CREDENTIALS=/gcp/creds.json \
+  -it \
+  molt fetch ...
```

Comment:
I'm not sure how picky Docker is but instead of `molt`, more accurately it's the container name, which should be `cockroachdb/molt` and potentially `cockroachdb/molt:latest` to be completely accurate. `molt` was just the name of my local docker image.

Reply:
Global comment for each example where we do `molt fetch` and Docker

Reply:
You mean `molt fetch` -> `cockroachdb/molt:latest fetch`, right?

Reply:
yep. Can probably just do `cockroachdb/molt`